src/main/java/com/infamous/combat_next/util/CombatUtil.java

```diff
@@ -170,7 +173,7 @@ private static void sweepAttack(Player player){
         float attackDamage = (float)player.getAttributeValue(Attributes.ATTACK_DAMAGE);
         float damageBonus = EnchantmentHelper.getDamageBonus(player.getMainHandItem(), MobType.UNDEFINED);
 
-        float attackStrengthScale = player.getAttackStrengthScale(0.5F);
+        float attackStrengthScale = MeleeCombatConfigs.getAttackCooldownImpactOnDamage().get() ? player.getAttackStrengthScale(0.5F) : 1;
```

Comment:
This config should be named in a way that specifies it is for missed attack sweep damage (however you can shorten that).

Reply:
it isn't, though, it is also used for a standard attack where you do not miss